src/general-components/Panel/index.js

```diff
@@ -0,0 +1,3 @@
+import Div from './Panel'
```

Comment:
For consistency, when importing what is default from a file, just call it the same as the file
```
import Panel from './Panel'
```